protocol/javatests/org/openyolo/protocol/HintRetrieveRequestTest.java

```diff
@@ -36,6 +38,77 @@
 public class HintRetrieveRequestTest {
 
     @Test
+    public void testFromAuthMethods() {
+        HintRetrieveRequest request = HintRetrieveRequest.fromAuthMethods(
+                AuthenticationMethods.EMAIL,
+                AuthenticationMethods.FACEBOOK);
+
+        assertThat(request.getAuthenticationMethods().size()).isEqualTo(2);
+        assertThat(request.getAuthenticationMethods()).contains(AuthenticationMethods.EMAIL);
```

Comment:
FYI, I think there is a containsExactly() method for set equality assertions.

Reply:
Oooh, me gusta. Done.